src/window-event-handler.coffee

```diff
@@ -141,8 +141,8 @@ class WindowEventHandler
       @atomEnvironment.hide()
     @reloadRequested = false
 
-    @atomEnvironment.storeDefaultWindowDimensions()
     @atomEnvironment.storeWindowDimensions()
+    @atomEnvironment.saveState()
```

Comment:
Ah yeah. https://github.com/atom/atom/pull/10605 must have broken that in the case where you reloaded the window without ever interacting with it and going idle afterwards. This should fix that.